source/documentation/adrs/adr-007.html.md.erb

```diff
@@ -0,0 +1,46 @@
+---
+title: ADR-007 Healthcheck Definitions
+last_reviewed_on: 2023-03-21
+review_in: 12 months
+---
+# <%= current_page.data.title %>
+
+### Date: 14-02-2023
+
+## Status
+
+✅ Accepted
+
+## Context
+
+Across our teams there are currently slight differences in how we use the various health checks within services. 
+
+This is mainly in the fidelity of the checks and what they connect to. For example one service might use a health check to just check if a service's containers are running. Another might check that in-app connectivity is working, such as front end to internal API to DB. Another might check that the service can connect out to related services it depends on, such as Notify or other API endpoints.
+
+This means that an engineer moving between products, either for temporary support or moving into a new role, could be confused at the intent of the healthcheck or make assumptions based on past experience that turn out not to be true. This is particularly tru if the healthcheck is used for multiple purposes which could be at odds, for example if a healthcheck connecting to an external service was used to cause containers to scale or cycle when they didn't need to. 
+
+## Decision
+
+We need to be clear on the purpose of each healthcheck within a service and be consistent across services in how we name, use and check them. Therefore we should categorise them as: container health checks, service health checks and dependency health checks so we are clear abotu what type we implement and how they are intended to be used.
+
+### Container Health Checks
+
+These are used to ascertain if an individual container is running as expected.
```

Comment:
I've been using the term ready to handle requests

Reply:
You're right, "as expected" is doing a lot of heavy lifting in that sentence.